packages/react-docs-example/public/css/basscss_responsive_type_scale.css

```diff
@@ -1,84 +0,0 @@
-/* Basscss Responsive Type Scale */
-/* Modified by Fabio Berger to include xs prefix */
```

Comment:
Why delete all these files?

Reply:
when I created the `react-docs-example` package, we had just separated out the docs rendering code from the website and were under the impression that others would want to use it. This didn't pan out. Additionally, I just don't feel confident given this most recent refactor that this package is ready for prime-time. We can always add it back once it is, but right now, I'd rather not give the impression that it's ready.